jwst/master_background/master_background_mos_step.py

```diff
@@ -106,9 +107,10 @@ def process(self, data):
             # If some type of background processing had already been done. Abort.
             # UNLESS forcing is enacted.
             if not self.force_subtract and \
-               'COMPLETE' in [data_model.meta.cal_step.back_sub, data_model.meta.cal_step.master_background]:
+               'COMPLETE' in [query_step_status(data_model, "back_sub"), \
```

Comment:
Following up on https://github.com/spacetelescope/jwst/pull/8600#discussion_r1661358917

Thanks. It would be helpful to see where this receives a container. I don't think any of the containers provide `meta.cal_step` so I would expect the old code to fail with `AttributeError: No attribute 'cal_step'` when a container is provided.

Reply:
it did indeed fail.  I will track this down and let you know

Reply:
I couldn't figure out where the failures were in previous iterations of this, if any - I think I have thoroughly confused myself about this.  I started [yet another regression test](https://plwishmaster.stsci.edu:8081/job/RT/job/JWST-Developers-Pull-Requests/1574/) with the change to `master_background_mos` reverted to see if any failures do crop up.  Either way, I think I'm comfortable with the current `query_step_status`

Reply:
yes indeed, I was wrong again - this turned up no failures.